bin/ocaml/ocaml_merlin.ml

```diff
@@ -99,7 +99,7 @@ end = struct
     | None ->
       Path.Build.drop_build_context_exn file
       |> Path.Source.to_string_maybe_quoted
-      |> Printf.sprintf "No config found for file %s. Try calling `dune build`."
+      |> Printf.sprintf "No config found for file %s. Try calling 'dune build'."
```

Comment:
Why not use `User_message.command` here?

Reply:
Because it gets used as input to `Merlin_conf.make_error` which does not get raised as `User_error.raise` does, but instead serialized to an S-Expression (and probably transmitted somewhere, and I didn't want to do the research of what breaks if we send formatted data there).